Benchmark/cmake/modules/Benchmark.cmake

```diff
@@ -41,6 +41,7 @@ function(add_benchmark CHECKID)
   target_link_libraries(${CHECKID}
     PUBLIC
       benchmark::benchmark
+      m
```

Comment:
I doubt this would work on non Unix systems. The [correct approach](https://stackoverflow.com/a/74867749) would be:

```
find_library(MATH_LIBRARY m)
if(MATH_LIBRARY)
    target_link_libraries(MyTarget PUBLIC ${MATH_LIBRARY})
endif()
```

Reply:
Should be fixed after rebasing with `main`! :)

Reply:
TYSM @alvrogd !